keyboards/handwired/dactyl_manuform/4x5/keymaps/kombo/keymap.c

```diff
@@ -0,0 +1,211 @@
+/* auth: domonkos */
+
+#include "dactyl_manuform.h"
+#include "mousekey.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.